src/test/test.spec.js

```diff
@@ -0,0 +1,235 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import { describe, it } from 'mocha';
+import app from '../app';
+
+chai.use(chaiHttp);
+
+describe('SendIT API TESTS', () => {
+  // Test Invalid pathss
+  describe('/GET INVALID_PATH', () => {
```

Comment:
This test doesn't add any value to the project. Consider removing it.

Reply:
Yes I Removed It